src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -80,13 +80,10 @@
   "commands.neoforge.setdim.invalid.dim": "The dimension ID specified (%1$s) is not valid.",
   "commands.neoforge.setdim.invalid.nochange": "The entity selected (%1$s) is already in the dimension specified (%2$s).",
   "commands.neoforge.setdim.deprecated": "This command is deprecated for removal in 1.17, use %s instead.",
-  "commands.neoforge.tps.invalid": "Invalid dimension %1$s Possible values: %2$s",
-  "commands.neoforge.tps.summary.overall": "Overall:",
-  "commands.neoforge.tps.summary.dimension": "'%s':",
-  "commands.neoforge.tps.mean_tps": "Mean TPS: %s",
-  "commands.neoforge.tps.mean_tps.tooltip": "Closer to 20 is better",
-  "commands.neoforge.tps.mean_tick_time": "Mean tick time: %s ms",
-  "commands.neoforge.tps.mean_tick_time.tooltip": "Lower is better, Higher is worse",
+  "commands.neoforge.tps.overall": "Overall: %s TPS (%s ms/tick)",
+  "commands.neoforge.tps.tooltip": "Mean TPS; higher is better. Target TPS: %s",
+  "commands.neoforge.tps.dimension": "%s: %s TPS (%s ms/tick)",
+  "commands.neoforge.tps.dimension.tooltip": "%s (%s)",
```

Comment:
As I understand from the code above, this tooltip contains the location of the dimension, and the location of the dimension _type_ in the parentheses. I didn't understand that on first glance, and I fear it might become a point of confusion for other users. 

Should we try to clarify what the location in parentheses means (the dimension _type_), or do we assume the user knows what it means?

Reply:
Looking at vanillas own translations they do include some context with the translation key.
Potions being a good example `"potion.withDuration": "%s (%s)"`.

I could update the key from `commands.neoforge.tps.dimension.tooltip` to `commands.neoforge.tps.dimension.withType` or something similar?

This might also be better suited as an extension for https://github.com/neoforged/NeoForge/pull/1428.
To add dimension type translation lookups. [`dimension_type.<namespace>.<path>`]
would also add simple utility translation
`"dimension.withType": "%s (%s)"` -> `<dimension component> (<dimension type component>)`

Reply:
Just to be clear, I wasn't talking more about the translation key, but more on how it's shown in game: `minecraft:overworld (minecraft:overworld)` (for example) seems a bit odd to someone using the command that doesn't know the distinction between the two.

(It's acceptable with the vanilla potion duration translation, because the first value is the potion name and the second is a duration; both can't be confused for the other.)

Reply:
Oh. Yeah the format might be confusing for a regular user. However since the concept of TPS and the details of this command are intended for more intermediate users, I think its reasonable to assume they know the distinction between the 2.

Reply:
Okay then. If it becomes a reported problem, we can probably just insert "dimension type: " then to clarify.

Reply:
Please do this. I would call myself a "power user" who has been playing Minecraft since beta 1.4, but this is the first time I've come across something like dimension types. Perhaps I'm just ignorant, which is very likely :)

Reply:
Please do this if possible. I would call myself a "power user" who has been playing Minecraft since beta 1.4, but this is the first time I've come across something like dimension types. Perhaps I'm just ignorant, which is very likely :)

Reply:
Appended `Dimension Type:` to the tooltip text, to make this distinction more obvious.

![image](https://github.com/user-attachments/assets/650e04a2-522e-4772-a490-e68191374a1e)